recipes/example/meta.yaml

```diff
@@ -1,82 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
```

Comment:
Please, do not touch this file.
Could you please revert it?

Reply:
No problem. Rebased the commit to include it.